systemtests/src/test/java/io/enmasse/systemtest/bases/web/ConsoleTest.java

```diff
@@ -219,13 +219,73 @@ protected void doTestCreateAddrSpaceNonClusterAdmin() throws Exception {
         int addressCount = 4;
         String namespace = "test-namespace";
         UserCredentials user = Credentials.userCredentials();
+        KubeCMDClient.createNamespace(namespace);
+        kubernetes.getClient().rbac().clusterRoles().createOrReplaceWithNew()
+                .editOrNewMetadata()
+                .withName(namespace)
+                .endMetadata()
+                .addNewRule()
+                .withApiGroups("")
+                .withResources("namespaces")
+                .withVerbs("get")
+                .withResourceNames(namespace)
+                .endRule()
+                .done();
+
+        kubernetes.getClient().rbac().clusterRoleBindings().createOrReplaceWithNew()
+                .editOrNewMetadata()
+                .withName(namespace)
+                .endMetadata()
+                .addNewSubject()
+                .withApiGroup("rbac.authorization.k8s.io")
+                .withKind("User")
+                .withName("pepa")
+                .endSubject()
+                .editOrNewRoleRef()
+                .withApiGroup("rbac.authorization.k8s.io")
+                .withKind("ClusterRole")
+                .withName(namespace)
+                .endRoleRef()
+                .done();
+
+        kubernetes.getClient().rbac().clusterRoleBindings().createOrReplaceWithNew()
+                .editOrNewMetadata()
+                .withName(namespace)
+                .endMetadata()
+                .addNewSubject()
+                .withApiGroup("rbac.authorization.k8s.io")
+                .withKind("User")
+                .withName("pepa")
+                .endSubject()
+                .editOrNewRoleRef()
+                .withApiGroup("rbac.authorization.k8s.io")
+                .withKind("ClusterRole")
+                .withName(namespace)
+                .endRoleRef()
+                .done();
+
+        kubernetes.getClient().rbac().roleBindings().inNamespace(namespace).createOrReplaceWithNew()
```

Comment:
Should this be referring to a role rather than a cluster role?

Reply:
You can refer to a clusterrole, and it will apply to the namespace of the rolebinding only. If creating a clusterrolebinding, it will apply to all namespaces.